src/views/layouts/main.php

```diff
@@ -116,16 +116,16 @@
                                 <span class="mainnav-tooltip-big">
                                     <ul class="mainnav-tooltip-big-menu">
                                         <li class="mainnav-tooltip-big-menu-item" ui-sref-active="mainnav-tooltip-big-menu-item-active" ui-sref="custom({templateId:'admin/account/dashboard'})" ng-click="isOpen=0">
-                                            <a class="mainnav-tooltip-big-menu-link">
+                                            <span class="mainnav-tooltip-big-menu-link">
```

Comment:
If I read everything correctly (didn't check it locally) the class will never be used because the scss generates the class `.mainnav-tooltip-big-menu-item-link` because of the nesting.

Reply:
Yes. you might be correct. let check and correct it.